android/src/main/java/io/sentry/RNSentryModule.java

```diff
@@ -190,4 +219,114 @@ private Level logLevel(int level) {
                 return Level.OFF;
         }
     }
+
+    @ReactMethod
+    public void setUser(final ReadableMap user) {
+        if (user == null) {
+            Sentry.setUser(null);
+        } else {
+            User userInstance = new User();
+
+            // only support these defined keys for now.
+            if (user.hasKey("email")) {
+                userInstance.setEmail(user.getString("email"));
+            } else {
+                userInstance.setEmail(null);
```

Comment:
why do we explicitly set `null` here when it's a newly created instance?
Same for the following lines

Reply:
🙃

Reply:
Thank you for telling me that completely went over my head that it would be null by default.